arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -696,6 +696,46 @@ def rule_gfm_1_3_1(
         )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_3_2(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.3.2] The schemaLocation attribute of the xsd:import element specifying the EDINET taxonomy does not point to a valid location.
+
+    GFM-1.3.2:  If an xsd:import element has a namespace attribute equal to a standard taxonomy
+    schema, then its schemaLocation attribute must be the standard taxonomy assigned
+    to that namespace.
+    """
+    for document in val.modelXbrl.urlDocs.values():
+        if not pluginData.isExtensionUri(document.uri, val.modelXbrl) or not document.type == ModelDocument.Type.SCHEMA:
+            continue
+        for refDoc in document.referencesDocument.values():
+            if 'import' not in refDoc.referenceTypes:
+                continue
+            namespace = refDoc.referringModelObject.attrib.get('namespace')
+            if not namespace:
+                continue
+            if namespace.endswith('iod'):
+                namespace = namespace[:-3]
+            if namespace.startswith(EDINET_NAMESPACE_PREFIX):
+                schemaLocation = refDoc.referringModelObject.attrib.get('schemaLocation')
+                if schemaLocation and not schemaLocation.startswith(namespace):
```

Comment:
It seems like we should be checking `schemaLocation == namespace + ".xsd"` to reject cases where the `schemaLocation` is valid but contains unwanted trialing characters (for example, a query parameter).

That only works if EDINET namespace URIs and XSD URLs are consistent enough.  Otherwise, I guess we need a full table of namespace/XSD pairs like EDGAR?  I wonder whether `http` vs `https` should be tolerated now or will need to be in the future.